### PR TITLE
[full-ci] [tests-only] Use sharing-ng in given steps for sharing

### PR DESCRIPTION
**Pull request by grgprarup**
## Description
Used sharingNG graph API in the given steps for sharing.

Suites covered:
- `apiLocks/unlockFiles.feature`
- `apiReshare/resharing.feature`
- `apiSpaces/tag.feature`
- `apiSpacesShares/`
   - `copySpaces.feature`
   - `shareSpaces.feature`
   - `createFileFolderWhenSharesSpaceExist.feature`
- `coreApiShareManagementToShares/acceptShares.feature`
- `coreApiShareOperationsToShares2/`
  - `getWebDAVSharePermissions.feature`
  - `uploadToShare.feature`
- `coreApiShareUpdateToShares/updateShare.feature`
- `coreApiWebdavUpload/uploadFile.feature`
- `coreApiWebdavUploadTUS/`
  - `uploadFileMtimeShares.feature`
  - `uploadToNonExistingFolder.feature`
  - `uploadToShare.feature`
- `parallelDeployAcceptance/features/apiShareManagement/acceptShares.feature`

## Related Issue

- Part of https://github.com/owncloud/ocis/issues/8717

## How Has This Been Tested?

- Locally
- :robot: 

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests only (no source changes)

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link>

### CODE REVIEW

File: tests/acceptance/features/apiSpacesShares/createFileFolderWhenSharesSpaceExist.feature [hunk 1]
```diff
@@ -9,7 +9,12 @@ Feature: create file or folder named similar to Shares folder
       | Alice    |
       | Brian    |
     And user "Alice" has created folder "/FOLDER"
-    And user "Alice" has shared folder "/FOLDER" with user "Brian" with permissions "read,update"
+    And user "Alice" has sent the following resource share invitation:
+      | resource        | FOLDER   |
+      | space           | Personal |
+      | sharee          | Brian    |
+      | shareType       | user     |
+      | permissionsRole | Viewer   |
```
Comment (nabim777) [inline tests/acceptance/features/apiSpacesShares/createFileFolderWhenSharesSpaceExist.feature:17]:
```suggestion
      | permissionsRole | Editor   |
```
Is not `read,update` permission means Editor

**Review by saw-jan — APPROVED** [review]

File: tests/acceptance/features/apiSpacesShares/createFileFolderWhenSharesSpaceExist.feature (hunk 1 above)
Comment (grgprarup) [reply tests/acceptance/features/apiSpacesShares/createFileFolderWhenSharesSpaceExist.feature:17]: `read+update` permissions for a file will be `File Editor` role. For `Editor` role of a folder we need `read+update+create+delete` permissions.

File: tests/acceptance/features/apiReshare/resharing.feature [hunk 2]
```diff
@@ -140,4 +151,4 @@ Feature: re-share resources
       | 27          | share + view + edit + delete          |
       | 29          | share + view + create + delete        |
       | 31          | share + view + create + edit + delete |
-      
+
```
Comment (nabim777) [inline tests/acceptance/features/apiReshare/resharing.feature:154]:
```suggestion
```

Comment (saw-jan) [reply tests/acceptance/features/apiReshare/resharing.feature:154]: this is good. we should keep the newline

**Review by saw-jan — APPROVED** [review]

**Pull request merged**